src/jqLite.js

```diff
@@ -55,7 +55,7 @@
  * - [`children()`](http://api.jquery.com/children/) - Does not support selectors
  * - [`clone()`](http://api.jquery.com/clone/)
  * - [`contents()`](http://api.jquery.com/contents/)
- * - [`css()`](http://api.jquery.com/css/) - Only retrieves inline-styles, does not call `getComputedStyle()`
+ * - [`css()`](http://api.jquery.com/css/) - Only retrieves inline-styles, does not call `getComputedStyle()`. On write, does not convert numbers to strings and append 'px'.
```

Comment:
I fixed the wording but not the line length. Next time!

----- Ursprüngliche Nachricht -----
Von: "Michał Gołębiowski" <notifications@github.com>
Gesendet: ‎27.‎04.‎2015 22:35
An: "angular/angular.js" <angular.js@noreply.github.com>
Cc: "Martin Staffa" <mjstaffa@googlemail.com>
Betreff: Re: [angular.js] docs(angular.element): css() api incompatibility.(#11614)

In src/jqLite.js:
> @@ -55,7 +55,7 @@
>   * - [`children()`](http://api.jquery.com/children/) - Does not support selectors
>   * - [`clone()`](http://api.jquery.com/clone/)
>   * - [`contents()`](http://api.jquery.com/contents/)
> - * - [`css()`](http://api.jquery.com/css/) - Only retrieves inline-styles, does not call `getComputedStyle()`
> + * - [`css()`](http://api.jquery.com/css/) - Only retrieves inline-styles, does not call `getComputedStyle()`. On write, does not convert numbers to strings and append 'px'.
Also, this line is too long, it should be broken.
—
Reply to this email directly or view it on GitHub.